samcli/lib/cookiecutter/interactive_flow_creator.py

```diff
@@ -0,0 +1,125 @@
+""" This module parses a json/yaml file that defines a flow of questions to fulfill the cookiecutter context"""
+from typing import Dict, Optional, Tuple
+import yaml
+from samcli.commands.exceptions import UserException
+from samcli.yamlhelper import parse_yaml_file
+from .interactive_flow import InteractiveFlow
+from .question import Question, QuestionKind
+
+
+class QuestionsNotFoundException(UserException):
+    pass
+
+
+class QuestionsFailedParsingException(UserException):
+    pass
+
+
+class InteractiveFlowCreator:
+    @staticmethod
+    def create_flow(flow_definition_path: str, extra_context: Optional[Dict] = None):
```

Comment:
One thing to consider for the future. this flow definition path is static, however there are parts of `sam init` specifically those deal with event bridge, where the questions are generated dynamically based on the previous option selected.

Can we chain together flows in the future?